pkg/flags/flags.go

```diff
@@ -34,6 +38,13 @@ func setLogLevel(logLevel string) {
 		log.Fatalf("invalid log-level: %s", logLevel)
 	}
 	log.SetLevel(level)
+
+	klog.SetOutput(ioutil.Discard)
+	// Anything lower than the INFO level according to log is sent to /dev/null
+	if level == log.DebugLevel {
+		// Set stderr to INFO severity see https://github.com/kubernetes/klog/issues/23
+		klog.SetOutputBySeverity("INFO", os.Stderr)
```

Comment:
Just `-stderrthreshold` to `info` and that's good enough.

Reply:
Same deal here as my previous comment pointed out.